FreeRTOS-Plus/Demo/Common/Logging/windows/Logging_WinSim.c

```diff
@@ -286,17 +286,19 @@ void vLoggingPrintf( const char * pcFormat,
             pcTaskName = pcNoTask;
         }
 
-        if( strcmp( pcFormat, "\r\n" ) != 0 )
+        if( ( xAfterLineBreak == pdTRUE ) && ( strcmp( pcFormat, "\r\n" ) != 0 ) )
```

Comment:
This code appears functionally identical to the previous version.  `xAfterLineBreak` is always true here, it was initialized on line 265 and hasn't changed since then.

Reply:
It is a static declaration. Its value is modified below.

Reply:
aha missed that, thank you!